app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "network-api",
+  "description": "API and CMS for The Mozilla Network site",
+  "addons": [
+    "heroku-postgresql"
+  ],
+  "scripts": {
+    "postdeploy": "python app/manage.py migrate"
+  }
```

Comment:
Can you elaborate what you mean by add the env?
Can we not just have review apps inherit their environment variables from staging?